_posts/2019-10-29-dhis2-and-docker.md

```diff
@@ -0,0 +1,214 @@
+---
+title: DHIS2 with docker
+layout: post
+categories: [blog, docker]
+author: gintare
+---
+
+# DHIS2 with docker
```

Comment:
Since you have a `title:` in the frontmatter above, you don't need to duplicate the title here, in fact, it is better not to because the first paragraph will be used as the text snippet on the front page.

And, this allows you to use the primary heading in the text for sections so you can change all `##` to `#` and all `###` to `##`, etc.